src/models/dataStore.model.js

```diff
@@ -0,0 +1,38 @@
+const BaseModel = require('./base.model')
+const ApplicationData = require('../persistence/entities/applicationData.entity')
+
+module.exports = class DataStore extends BaseModel {
```

Comment:
This class doesn't really seem to be modelling anything in the business layer and is pretty much just a wrapper around the the underlying entity.

Is it needed at this point or could we just create business models as required that use the entity?

Reply:
We'll look into this later